lib/renderer/api/remote.js

```diff
@@ -219,7 +214,7 @@ function metaToValue (meta) {
   }
 
   if (meta.type in types) {
-    types[meta.type]()
+    return types[meta.type]
```

Comment:
This is kind of a major bug fixed here... didn't the prevous draft trigger any test failures?

Everyone makes typos and that's no big deal, but if we don't have unit tests that catch missing retvals, that's concerning...

Reply:
so, we don't actually have specific tests for this file...

Reply:
which isn't ideal in itself, i was previously just running the whole suite and seeing if i triggered previously passing tests in other areas to fail